### DISCUSSION TITLE
Can I change the exit shortcut?

### DISCUSSION THREAD

**Discussion by TinFoilFox** [fasteddy516/SimplySerial#29] [Ideas]
I've been using Simply Serial to connect to my Raspberry Pi headlessly but I've run into a problem.
I often use nano to edit text files but the shortcut to close the editor is `CTRL-X` which is the same as Simply Serial. This means that if I open a file with nano I cannot close it. `CTRL-X` will close the serial connection but when I reconnect, nano is still open, leaving my session stuck with an open editor and no way to close it.

If possible I would like to change the shortcut to close Simply Serial, maybe just add a modifier so it could be maybe something like `CTRL-ALT-X`?

But what do you think? Is there a solution I'm just not seeing?

**Comment by BlueBarks** [comment]
Ever find a solution? I came here to ask the same thing.

**Reply by TinFoilFox** [reply]
Yes. I stopped using nano to edit files 😂 . Not the solution I wanted but it was the only one that worked.

**Comment by BlueBarks** [comment]
😕

On Wed, Feb 26, 2025 at 12:46 PM Harrison Fry ***@***.***>
wrote:

> Yes. I stopped using nano to edit files 😂 . Not the solution I wanted but
> it was the only one that worked.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/fasteddy516/SimplySerial/discussions/29#discussioncomment-12329218>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACDYS3QTHDQR3IGZMWGSUE32RX4WDAVCNFSM6AAAAABX52UJ62VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTEMZSHEZDCOA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> com>
>


-- 
*Loran Bruce*
*1926 Terry Mill Road SE*
*Atlanta, GA 30316*
*678-951-4151: Cell*

**Reply by TinFoilFox** [reply]
It looks like the the author is working on something that might help. We might be able to change the exit key to something else in the next version.
Take a look at commit ab374f5084b4d58b676ee8f9ea51a5f3cfca3d00

**Reply by TinFoilFox** [reply]
Looks like it got pushed and you can use the `-exitkey` argument to change the letter from x to whatever we want!

**Comment by BlueBarks** [comment]
Thanks for the heads up! This is awesome!

*Loran Bruce*
*1926 Terry Mill Road SE*
*Atlanta, GA 30316*
*678-951-4151: Cell*
